object_storage/azure_container_handler.py

```diff
@@ -61,3 +61,20 @@ async def upload_binary_file(self, local_file_path: str, azure_path: str):
             blob_client = self.blob_container_client_async.get_blob_client(blob=azure_path)
             await blob_client.upload_blob(file, blob_type="BlockBlob", overwrite=True)
             Logger.logger.info(f'Finish to upload video for path: {local_file_path}')
+
+    async def save_file_to_local_path(self, os_path: str, local_path: str):
+        Logger.logger.info(f'Start to download file to: {local_path}')
+        with open(local_path, 'wb+') as file:
+            bytes = await self.get_binary_blob_context(os_path)
+            file.write(bytes)
+
+        Logger.logger.info(f'Finish to download file to: {local_path}')
+
+    async def get_binary_blob_context(self, os_path):
+        """
+        Get the blob context from azure.
```

Comment:
I am not sure about the word 'context',
You are returning here the blob content.